fenix/app/src/main/java/org/mozilla/fenix/home/topsites/TopSitesViewHolder.kt

```diff
@@ -31,23 +31,26 @@ class TopSitesViewHolder(
     @Composable
     override fun Content() {
         val topSites =
-            components.appStore.observeAsState(emptyList()) { state -> state.topSites }.value
+            components.appStore.observeAsComposableState { state -> state.topSites }.value
         val wallpaperState = components.appStore
-            .observeAsState(WallpaperState.default) { state -> state.wallpaperState }.value
+            .observeAsComposableState { state -> state.wallpaperState }.value
+            ?: WallpaperState.default
```

Comment:
Did this one need to be changed? To my knowledge, `observeAsState` was added to help provide a default value more easily.

Reply:
`observeAsState` is a fairly new API that was added by Rahul. I would expect adoption to be the low side.

Reply:
Ah I see, thanks for the context!